docs/public/_redirects

```diff
@@ -368,15 +368,21 @@ https://v4.material-ui.com/* https://v4.mui.com/:splat 301!
 
 # Proxies
 
-## Store
+## MUI Store
 /store/* https://mui-store.netlify.app/:splat 200
 /store-staging/* https://master--mui-store.netlify.app/:splat 200
 
 ## MUI X
 ## Unlike the store that expect to be hosted under a subfolder,
 ## MUI X is configured to be hosted at the root.
-/_next/* https://docs-v5--material-ui-x.netlify.app/_next/:splat 200
-/static/x/* https://docs-v5--material-ui-x.netlify.app/static/x/:splat 200
-/x/* https://docs-v5--material-ui-x.netlify.app/x/:splat 200
-/r/x-* https://docs-v5--material-ui-x.netlify.app/r/x-:splat 200
-/:lang/x/* https://docs-v5--material-ui-x.netlify.app/:lang/x/:splat 200
+/_next/* https://material-ui-x.netlify.app/_next/:splat 200
+/static/x/* https://material-ui-x.netlify.app/static/x/:splat 200
+/x/* https://material-ui-x.netlify.app/x/:splat 200
+/r/x-* https://material-ui-x.netlify.app/r/x-:splat 200
+/:lang/x/* https://material-ui-x.netlify.app/:lang/x/:splat 200
+
+## MUI Toolpad
+/_next/* https://mui-toolpad-docs.netlify.app/_next/:splat 200
```

Comment:
Out of curiosity, did we try Next.js `basePath` before?

Reply:
We have tried this pattern to do the integration with MUI Store and Gatsby: https://github.com/mui/mui-store/blob/67f30e5ec9a48aa229b605040c566c58a1d75a0c/web/gatsby-config.js#L17.

It simplifies the configuration on this file, however, we have to always do the mental conversion in our head between the localhost & live preview URLs on what will be the final URLs. I wouldn't say it has been a great DX so far on the store.